flink-filesystem-join-redis/README.md

```diff
@@ -19,7 +19,16 @@ following steps:
    - item_id, unique identifier of the item.
    - brand, the brand of this item.
    - category, the category of the item.
-   - timestamp, time when the item is created or updated.
+   - production_place, the location where the item is produced. It is a map with
+     the following entries.
+      - country, the country where the item is produced.
+      - city, the city where the item is produced.
+   
+   During the joining, the brand and production city would be appended to each
```

Comment:
It seems better to replace "production city" with something that explicitly mentions `production_place`.